anda/tools/F-Sy-H/F-Sy-H.spec

```diff
@@ -0,0 +1,31 @@
+Name:    F-Sy-H
+Version: 1.67
+Release: 1%?dist
+
+Summary: Feature-rich Syntax Highlighting for Zsh
+License: BSD-3-Clause
+URL:     https://github.com/z-shell/%name
+Source0: %url/archive/refs/tags/v%{version}.tar.gz
+
+BuildArch: noarch
+
+BuildRequires: zsh
```

Comment:
You can delete this, there isn't a build script anyways :p

Reply:
I think those are supposed to be deletions